src/aws/roles.tf

```diff
@@ -102,6 +102,35 @@ resource "aws_iam_role_policy_attachment" "lambda_pass_role_policy_attachment" {
   policy_arn = "${aws_iam_policy.lambda_pass_role_policy.arn}"
 }
 
+resource "aws_iam_policy" "lambda_s3_access_policy" {
+  name        = "${terraform.workspace}-marsha-lambda-s3-access-policy"
+  path        = "/"
+  description = "IAM policy to read in source bucket and write in destination bucket"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": ["s3:GetObject"],
+      "Effect": "Allow",
+      "Resource": "arn:aws:s3:::${aws_s3_bucket.marsha_source.bucket}/*"
+    },
+    {
+      "Action": ["s3:PutObject"],
+      "Effect": "Allow",
+      "Resource": "arn:aws:s3:::${aws_s3_bucket.marsha_destination.bucket}/*"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy_attachment" "lambda_s3_access_policy_attachment" {
+  role        = "${aws_iam_role.lambda_invocation_role.name}"
+  policy_arn  = "${aws_iam_policy.lambda_s3_access_policy.arn}"
```

Comment:
add comment why lambda needs this access? 
I think we should list the reasons in comments for each policy we grant. It will help when we will try to figure out why each policy is there ans whether it is still for a good reason.